src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.iOS/AppleCertificatePal.ImportExport.cs

```diff
@@ -118,6 +118,15 @@ internal static X509ContentType GetDerCertContentType(ReadOnlySpan<byte> rawData
                 // We ignore keyStorageFlags which is tracked in https://github.com/dotnet/runtime/issues/52434.
                 // The keys are always imported as ephemeral and never persisted. Exportability is ignored for
                 // the moment and it needs to be investigated how to map it to iOS keychain primitives.
```

Comment:
I think we can delete the TODO comment since we've now "to-done" something.  It could be augmented to say that we can do better, but the comment as it stands is incorrect with this change (we don't ignore the flags, we throw on the ones that might introduce complexity)

Reply:
Done!